changelog/16539.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
I agree this is a bug, but I wonder if we should label it `change` instead, since we're modifying the behaviour of the merge endpoint such that requests that succeeded before (whatever their untoward consequences) will now fail?

Reply:
I didn't know about `change` as an option, but knowing that it exists, I agree. It didn't feel appropriate to list as an improvement, obviously. Change feels right.